workers/unity/Packages/com.improbable.gdk.transformsynchronization/package.json

```diff
@@ -1,11 +1,11 @@
 {
-    "name": "com.improbable.gdk.transformsynchronization",
-    "displayName": "SpatialOS GDK Transform Synchronization",
-    "version": "0.2.0",
-    "unity": "2018.3",
-    "author": "Improbable Worlds Ltd",
-    "description": "SpatialOS GDK Transform Synchronization Module. Made by Improbable.",
-    "dependencies": {
-        "com.improbable.gdk.core":"0.2.0"
-    }
-}
+  "name": "com.improbable.gdk.transformsynchronization",
+  "displayName": "SpatialOS GDK Transform Synchronization",
+  "version": "0.2.1",
+  "unity": "2018.3",
+  "author": "Improbable Worlds Ltd",
+  "description": "SpatialOS GDK Transform Synchronization Module. Made by Improbable.",
+  "dependencies": {
+    "com.improbable.gdk.core": "0.2.1"
+  }
+}
```

Comment:
do we want a newline at the end?

Reply:
Probably - the release tool stripped all those newlines 😕